src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/CertificateRequest.cs

```diff
@@ -0,0 +1,463 @@
+                throw new ArgumentNullException(nameof(subjectName));
+            if (key == null)
+                throw new ArgumentNullException(nameof(key));
+            if (string.IsNullOrEmpty(hashAlgorithm.Name))
+                throw new ArgumentException(SR.Cryptography_HashAlgorithmNameNullOrEmpty, nameof(hashAlgorithm));
+
+            SubjectName = subjectName;
+
+            _key = key;
+            _generator = X509SignatureGenerator.CreateForRSA(key, RSASignaturePadding.Pkcs1);
+            PublicKey = _generator.PublicKey;
+            HashAlgorithm = hashAlgorithm;
+        }
+
+        /// <summary>
+        /// Create a CertificateRequest for the specified subject name, encoded public key, and hash algorithm.
+        /// </summary>
+        /// <param name="subjectName">
+        ///   The parsed representation of the subject name for the certificate or certificate request.
+        /// </param>
+        /// <param name="publicKey">
+        ///   The encoded representation of the public key to include in the certificate or certificate request.
+        /// </param>
+        /// <param name="hashAlgorithm">
+        ///   The hash algorithm to use when signing the certificate or certificate request.
+        /// </param>
+        public CertificateRequest(X500DistinguishedName subjectName, PublicKey publicKey, HashAlgorithmName hashAlgorithm)
+        {
+            if (subjectName == null)
+                throw new ArgumentNullException(nameof(subjectName));
+            if (publicKey == null)
+                throw new ArgumentNullException(nameof(publicKey));
+            if (string.IsNullOrEmpty(hashAlgorithm.Name))
+                throw new ArgumentException(SR.Cryptography_HashAlgorithmNameNullOrEmpty, nameof(hashAlgorithm));
+
+            SubjectName = subjectName;
+            PublicKey = publicKey;
+            HashAlgorithm = hashAlgorithm;
+        }
+
+        /// <summary>
+        /// Create an ASN.1 DER-encoded PKCS#10 CertificationRequest object representating the current state
+        /// of this object.
+        /// </summary>
+        /// <returns>A DER-encoded certificate signing request.</returns>
+        /// <remarks>
+        ///   When submitting a certificate signing request via a web browser, or other graphical or textual
+        ///   interface, the input is frequently expected to be in the PEM (Privacy Enhanced Mail) format,
+        ///   instead of the DER binary format. To convert the return value to PEM format, make a string
+        ///   consisting of <c>-----BEGIN CERTIFICATE REQUEST-----</c>, a newline, the Base-64-encoded
+        ///   representation of the request (by convention, linewrapped at 64 characters), a newline,
+        ///   and <c>-----END CERTIFICATE REQUEST-----</c>.
+        /// 
+        ///   <code><![CDATA[
+        ///     public static string PemEncodeSigningRequest(CertificateRequest request, PkcsSignatureGenerator generator)
```

Comment:
As a method with a different name I think it just confuses people that walk up to the class and don't know why they'd want PEM vs DER (or PEM vs "that default/other thing"); and the review process was suggesting a minimalistic API for the initial feature version anyways.  If you think it's too odd to have this type of helper method be in documentation, we can pull it.

But really, to me, this would belong in a bigger PEM-friendly API support feature.  (cert exporting, EnvelopedCms, SignedCms, this).